.travis.yml

```diff
@@ -5,11 +5,6 @@ git:
 dist: trusty
 sudo: required
 
-branches:
```

Comment:
there is also a piece in the [release readme](https://github.com/material-components/material-components-web-react/blob/master/docs/release.md#update-travisyml-config). Can you please update.